v2.0/resources/task.yml

```diff
@@ -67,3 +67,15 @@ allOf:
         format: date-time
         description: When the task object was created
         example: '2012-12-12T10:53:43-08:00'
+
+      completion_rule:
```

Comment:
Can this field also been set using any of the endpoints?

Reply:
Yes, on the POST / PUT endpoints as well - added completion_rule to both of those in new commit